README.md

```diff
@@ -570,22 +577,25 @@ Activate token quota limits for the service by adding a new configuration struct
       - name: user_monthly_limits
         type: user_limiter
         initial_quota: 100000 <3>
-        quota_increase: 10 <4>
         period: 30 days
       - name: cluster_monthly_limits
         type: cluster_limiter
-        initial_quota: 100000 <3>
-        quota_increase: 1000000 <4>
+        initial_quota: 1000000 <4>
         period: 30 days
+      - name: user_quota_daily_increases
+        type: user_limiter
+        quota_increase: 1000 <5>
+        period: 1 hour
```

Comment:
this period is not accepted
` spec.ols.quotaHandlersConfig.limitersConfig[0].period in body should match '^(1\s+(day|month|year|d|m|y)|([2-9][0-9]*|[1-9][0-9]{2,})\s+(days|months|years|d|m|y))$'"`

Reply:
user-wise, I don't see a scenario where a user would need a 1h quota reset or increase

Reply:
but ok, will change it to one day, if it makes sense